specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/DistributedAvailabilityGroups.json

```diff
@@ -0,0 +1,687 @@
+          "enum": [
+            "DISCONNECTED",
+            "CONNECTED"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ReplicaConnectedState",
+            "modelAsString": true
+          }
+        },
+        "lastReceivedLsn": {
+          "description": "Last received LSN",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastReceivedTime": {
+          "format": "date-time",
+          "description": "Last received LSN time",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastSentLsn": {
+          "description": "Last sent LSN",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastSentTime": {
+          "format": "date-time",
+          "description": "Last sent LSN time",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastCommitLsn": {
+          "description": "Last commit LSN",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastCommitTime": {
+          "format": "date-time",
+          "description": "Last commit LSN time",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastHardenedLsn": {
+          "description": "Last hardened LSN",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastHardenedTime": {
+          "format": "date-time",
+          "description": "Last hardened LSN time",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastBackupLsn": {
+          "description": "Last backup LSN",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastBackupTime": {
+          "format": "date-time",
+          "description": "Last backup LSN time",
+          "type": "string",
+          "readOnly": true
+        },
+        "mostRecentLinkError": {
+          "description": "The most recent link connection error description",
+          "type": "string",
+          "readOnly": true
+        },
+        "partnerAuthCertValidity": {
+          "$ref": "#/definitions/CertificateInfo",
+          "description": "SQL server certificate validity",
+          "readOnly": true
+        },
+        "instanceSendReplicationLagSeconds": {
+          "format": "int32",
+          "description": "Replication lag when Managed Instance link side is primary",
+          "type": "integer",
+          "readOnly": true
+        },
+        "instanceRedoReplicationLagSeconds": {
+          "format": "int32",
+          "description": "Redo lag when Managed Instance link side is primary",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "DistributedAvailabilityGroupProperties": {
+      "description": "The properties of a distributed availability group.",
+      "type": "object",
+      "properties": {
+        "distributedAvailabilityGroupName": {
```

Comment:
The 'distributedAvailabilityGroupName' already appears in the path, please don't repeat it in the request body.